modules/discretedata/channels/datachannel.h

```diff
@@ -43,11 +43,11 @@ typedef signed long long ind;
 *   If these do not suffice, cast the respective short.
 */
 enum class GridPrimitive : ind {
-    Undef = -1,
-    Vertex = 0,
-    Edge = 1,
-    Face = 2,
-    Volume = 3,
+    Undef       = -1,
```

Comment:
does clang-format give you this formatting? We always format everything with clang-format.
If you for some reason really don't want clang-format formating for a code section, that section as to be marked with
//clang-format off
section
//clang-format on

Reply:
We are kind of strict with the use of clang-format,

Reply:
But I can fix this later on. Have a bunch of minor things I will change...